charts/argo-cd/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: v2.2.3
+appVersion: v2.2.4
```

Comment:
Wrong version. Please revert `appVersion` and bump `version`.

Also please update the changelog on line 24